### PR TITLE
implemented route id

### PR DESCRIPTION
**Pull request by hyt152004**

### CODE REVIEW

**Comment by CraftinPark** [comment]
Task linked: [CU-86b0rn7m3 implement route id](https://app.clickup.com/t/86b0rn7m3)

File: src/classes/route.py [hunk 1]
```diff
@@ -2,10 +2,13 @@
 from .edge import Edge, get_length  
 
 class Route():
+    id: int                             = None           # vehicle identifier
```
Comment (changoug) [inline src/classes/route.py]: I think the spacing, while clean, will be harder to keep consistent across all the files we have. Also perhaps id should be generated by the class itself vs being a parameter that is passed in. A global iterator should suffice (every new vehicle increments the id by one), or perhaps some built-in identifier. I will leave that decision up to you.

File: src/classes/route.py [hunk 2]
```diff
@@ -2,10 +2,13 @@
 from .edge import Edge, get_length  
 
 class Route():
+    id: int                             = None           # vehicle identifier  
     edges: list[Edge]
-    total_length: float = 0
+    total_length: float                 = 0              # float representing total length of the route in meters
```
Comment (changoug) [inline src/classes/route.py]: Same thing with the spacing.

**Review by changoug — CHANGES_REQUESTED** [review]
Good stuff, just some comments on stuff I think that is worth discussing.

**Review by changoug — APPROVED** [review]
Good stuff!

**Pull request merged**